doc-changelog/action.yml

```diff
@@ -91,7 +99,49 @@ runs:
       run: |
         python -m pip install --upgrade pip towncrier==${{ inputs.towncrier-version }}
 
+    - name: "Get first letter of conventional commit type"
+      if: ${{ inputs.use-labels == 'false' }}
+      env:
+        PR_TITLE: ${{ github.event.pull_request.title }}
+      shell: python
+      run: |
+        import sys
+        sys.path.insert(1, '${{ github.action_path }}/../doc-changelog/')
+
+        from parse_pr_title import get_first_letter_case
+
+        pr_title = """${{ env.PR_TITLE }}"""
+        get_first_letter_case(pr_title)
+
+    - name: "Check pull-request title follows conventional commits style"
+      if: ${{ env.FIRST_LETTER == 'lowercase' }}
+      uses: ansys/actions/commit-style@v6
+      with:
+        token: ${{ inputs.token }}
+
+    - name: "Check pull-request title follows conventional commits style with upper case"
+      if: ${{ env.FIRST_LETTER == 'uppercase' }}
+      uses: ansys/actions/commit-style@v6
+      with:
+        token: ${{ inputs.token }}
+        use-upper-case: true
```

Comment:
If ``use-labels == 'true'`` we should skip all these steps, right?

Reply:
Or the same thing the other way around... these steps should only execute if ``use-labels == 'false'``